Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ContestsController.cs

```diff
@@ -225,7 +225,7 @@ public ActionResult GetLatestContests([DataSourceRequest]DataSourceRequest reque
             return this.PartialView(GlobalConstants.QuickContestsGrid, latestContests);
         }
 
-        public JsonResult GetCategories()
+        public JsonResult GetCategories(string contestFilter)
```

Comment:
Yes, the only place where the GetCategories method is called is in the CategoryDropDown.cshtml